go/fn/byteio.go

```diff
@@ -0,0 +1,58 @@
+package fn
```

Comment:
Suggest naming the file name to `io.go`.

Reply:
done